### PR TITLE
BACKLOG-12845 : use backend WIP service to read and store work in pro…

### PR DESCRIPTION
**Pull request by dgriffon**
https://jira.jahia.org/browse/BACKLOG-12845

### CODE REVIEW

File: src/javascript/EditPanel/WorkInProgress/Chip/WipInfo.Chip.spec.js [hunk 1]
```diff
@@ -34,13 +36,15 @@ describe('WipInfoChip', () => {
...
-        let wrapper = shallow(<WipInfoChip/>);
+        let RenderWrapper = shallow(<WipInfoChip/>).props().render;
+        let wrapper = shallow(<RenderWrapper field={field}/>);
         expect(wrapper.debug()).toContain('Chip');
     });
 
     it('Should display chip when showChipHeader is false', () => {
         showChip = false;
-        let wrapper = shallow(<WipInfoChip/>);
+        let RenderWrapper = shallow(<WipInfoChip/>).props().render;
```
Comment (FBerthelot) [inline src/javascript/EditPanel/WorkInProgress/Chip/WipInfo.Chip.spec.js]:
```suggestion
        const RenderWrapper = shallow(<WipInfoChip/>).props().render;
```

Comment (FBerthelot) [reply src/javascript/EditPanel/WorkInProgress/Chip/WipInfo.Chip.spec.js]: you can change that everywhere if you want :)

Comment (dgriffon) [reply src/javascript/EditPanel/WorkInProgress/Chip/WipInfo.Chip.spec.js]: let's stick to the current changes :)

File: src/javascript/Create/Create.adapter.js [hunk 2]
```diff
@@ -23,9 +24,12 @@ export const adaptCreateFormData = (data, lang, t) => {
     const nodeData = data.jcr.result;
     const sections = data.forms.createForm.sections;
 
+    // Work in progress
+    const wipInfo = {[Constants.wip.fieldName]: {status: Constants.wip.status.DISABLED, languages: []}};
```
[the remark this answers is not in the document]
Comment (dgriffon) [reply src/javascript/Create/Create.adapter.js]: I hesitated, bu as the getInitialValues was related to {{sections}} I kept it here .. it seems not to be an issue

**Pull request merged**